src/components/plots/rectanglePlot.ts

```diff
@@ -82,23 +74,32 @@ export module Plots {
     public x(x: number | Accessor<number>): Plots.Rectangle<X, Y>;
     public x(x: X | Accessor<X>, scale: Scale<X, number>): Plots.Rectangle<X, Y>;
     public x(x?: number | Accessor<number> | X | Accessor<X>, scale?: Scale<X, number>): any {
-      if (x == null) {
-        return super.x();
+
+      if (scale != null) {
+        var x2Binding = this.x2();
+        var x2 = x2Binding && x2Binding.accessor;
+        this._bindProperty(Rectangle._X2_KEY, x2, scale);
```

Comment:
Talked to @rcchen before starting this and he agreed, there was nothing he could think of where x and x2 / y and y2 would use different scales